napari_animation/interpolation.py

```diff
@@ -159,7 +164,10 @@ class Interpolation(Enum):
     SLERP = partial(slerp)
 
     def __call__(self, *args):
-        return self.value(*args)
+        # print(args)
+        val = self.value(*args)
+        # print(val)
+        return val
```

Comment:
```suggestion
        return self.value(*args)
```
looks like some debug prints got caught in here

Reply:
Ops